src/components/Dropdown/dropdown.module.scss

```diff
@@ -0,0 +1,60 @@
+@import '@/styles/mediaQueries.scss';
+@import '@/styles/variables.scss';
+@import '@/styles/colors.scss';
+
+.dropdown {
+  position: relative;
+  /* display: inline-block; */
+
+  .dropdownBtn {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    background-color: $white;
+    border: 1.5px solid $black;
+    border-radius: $brad-sm;
+    padding: 0.5rem;
+    padding-top: 0.75rem;
+
+    .arrowBtn {
+      width: 1.3rem;
+      height: 1.3rem;
+
+      &:hover {
+        cursor: pointer;
+      }
+    }
+  }
+
+  .radiusNull {
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
+  }
+
+  .dropdownContent {
+    position: absolute;
+    width: 100%;
+    height: 40rem;
+    overflow: scroll;
+    background-color: $white;
+    color: $black;
+    font-size: 0.85rem;
+    border: 1.5px solid $black;
```

Comment:
No pixel fractions, please, because of inconsistent rounding in different browsers:
https://www.simonbattersby.com/blog/browsers-and-fractional-pixels/

Reply:
oh no, really? blöd. sometimes, 2px is too much and 1 just too little. life..